pkg/antctl/raw/helper.go

```diff
@@ -220,3 +225,26 @@ func CreateControllerClientCfg(
 	cfg.Host = fmt.Sprintf("https://%s", net.JoinHostPort(nodeIP, fmt.Sprint(controllerInfo.APIPort)))
 	return cfg, nil
 }
+
+type PodFileCopy interface {
+	CopyFromPod(ctx context.Context, fs afero.Fs, namespace, name, containerName, srcPath, dstDir string) error
+}
+
+type PodFile struct {
+	RestConfig *rest.Config
+	Client     kubernetes.Interface
+}
+
+func (p *PodFile) CopyFromPod(ctx context.Context, fs afero.Fs, namespace, name, containerName, srcPath, dstDir string) error {
+	dir, fileName := filepath.Split(srcPath)
+	cmd := fmt.Sprintf("cd %s; tar cf - %s", dir, fileName)
+	if dir == "" {
+		cmd = fmt.Sprintf("tar cf - %s", fileName)
+	}
+	cmdArr := []string{"/bin/sh", "-c", cmd}
+	output, _, err := check.ExecInPod(ctx, p.Client, p.RestConfig, namespace, name, containerName, cmdArr)
```

Comment:
please move the `ExecInPod` function definition to this file, it will make more sense